docs/filter_expr.md

```diff
@@ -45,124 +45,123 @@ $cvss_v3_score 5 float >= $current_release_date 2023-12-31 timestamp > and
+| Operator    | Arguments             | Result                                                                                                    |
+| ----------- | --------------------- | --------------------------------------------------------------------------------------------------------- |
+| `true`      |                       | `true`                                                                                                    |
+| `false`     |                       | `false`                                                                                                   |
+| `not`       | `bool`                | `bool` negates argument                                                                                   |
+| `and`       | `bool` `bool`         | `bool` logical `and`s the two argments                                                                    |
+| `or`        | `bool` `bool`         | `bool` logical `or`s the two arguments                                                                    |
+| `float`     | `string` or `integer` | `float` Converts argument tp float                                                                        |
+| `integer`   | `string` or `float`   | `integer` Converts argument to integer                                                                    |
+| `timestamp` | `string`              | `timestamp` Converts argument to timestamp                                                                |
+| `workflow`  | `string`              | `workflow` Converts argument to workflow                                                                  |
+| `events`    | `string`              | `events` Converts argument to events                                                                      |
+| `=`         | **A** **B**           | `bool` **A** equals **B**                                                                                 |
+| `!=`        | **A** **B**           | `bool` **A** not equals **B**                                                                             |
+| `<`         | **A** **B**           | `bool` **A** lesser than **B\***                                                                          |
+| `<=`        | **A** **B**           | `bool` **A** lesser or equal than **B**                                                                   |
+| `>`         | **A** **B**           | `bool` **A** greater than **B**                                                                           |
+| `>=`        | **A** **B**           | `bool` **A** greater or equal than **B**                                                                  |
+| `ilike`     | `string` `string`     | `bool` First argument is case insensitive like second argument                                            |
+| `ilikepid`  | `string`              | `bool` Is there a product in the product tree with a name like the argument?                              |
```

Comment:
This seems to be the same as the original, so the problem is not fixed.
(Please do not make unrelated whitespace changes in a pull request. It makes reviews much harder.)